frontend/cypress/integration/projects/settings.spec.ts

```diff
@@ -68,6 +66,7 @@ describe('project settings', () => {
         );
 
         cy.get("[data-testid='ADD_VARIANT_BUTTON']").first().click();
+        cy.wait(300);
```

Comment:
I'd rather await on the element that appears asynchronously. 300ms wait is very arbitrary.